src/io/obf.rs

```diff
@@ -72,233 +72,179 @@ pub async fn obfuscate<M, SU, SH, ST, R, P>(
 
     Sample the initial public key (level 0) with our reveal flags.
 
-    Sample the initial BGG+ encodings.
-    **NOTE:** the paper treats
-        - c_att = encoding of (1, bits(X)), where X represents the evaluator's inputs.
-        - c_t = encoding of the FHE secret-key
-    as two distinct vectors.
-
-    The length of encodings_init is (1 + packed_input_size + 1):
-       - 1 for the encoding of 1
-       - packed_input_size for the packed evaluator inputs,
-       - 1 for the encoding of t (secret key).
-
-    encodings_init is bundled into a single Vec<bool> of length L,
-    where:
-       - indices 0..L-2 → c_att
-       - index   L-1   → c_t
+    The length of encodings_init is (1 + packed_input_size):
```

Comment:
As I mentioned in another comment, I think it is better to say that the initial 1 is for the encoding of 1 and the packed input size refers, in order, to the packed evaluator input bits and the RLWE secret key `t`